examples/high_level_api/fastapi_server.py

```diff
@@ -156,7 +156,7 @@ class CreateChatCompletionRequest(BaseModel):
     top_p: float = 0.95
     stream: bool = False
     stop: List[str] = []
-    max_tokens: int = 128
+    max_tokens: Optional[Union[int, None]] = -1
```

Comment:
The type signature for `max_tokens: Optional[Union[int, None]]` doesn't make any sense.
`Optional[T]` is equivalent/alias of `Union[T, None]`